ts/features/messages/components/MessageDetail/HeaderDueDateBar.tsx

```diff
@@ -1,45 +1,28 @@
-import * as React from "react";
+import React from "react";
 import { UIMessageDetails } from "../../types";
 import { getPaymentExpirationInfo } from "../../utils";
 import DueDateBar from "./DueDateBar";
-import MedicalPrescriptionDueDateBar from "./MedicalPrescriptionDueDateBar";
-import MedicalPrescriptionIdentifiersComponent from "./MedicalPrescriptionIdentifiersComponent";
 
-type Props = {
+type HeaderDueDateBarProps = {
   messageDetails: UIMessageDetails;
   hasPaidBadge: boolean;
 };
 
-/**
- * Groups all the possible combination of DueDate bars based (or not) on the prescriptionData
- * @param props
- * @constructor
- */
-export const HeaderDueDateBar = (props: Props): React.ReactElement => {
-  const paymentExpirationInfo = getPaymentExpirationInfo(props.messageDetails);
+export const HeaderDueDateBar = ({
+  hasPaidBadge,
+  messageDetails
+}: HeaderDueDateBarProps) => {
+  const paymentExpirationInfo = getPaymentExpirationInfo(messageDetails);
 
   return (
     <>
```

Comment:
Should we remove this fragment?

Reply:
Applied https://github.com/pagopa/io-app/pull/5407/commits/e46c41718336a23b016908d22e39f18cd64a177e